cmd/ansible-operator/main.go

```diff
@@ -81,7 +82,16 @@ func registerGVK(gvk schema.GroupVersionKind) {
 	k8sutil.AddToSDKScheme(schemeBuilder.AddToScheme)
 }
 
+// Used to set the decoder to just deserialize instead of decode.
+// This is only need as a work around for now and when the SDK is based on
+// controller-runtime we expect this to be fixed.
```

Comment:
Is there an issue tracking whatever this problem and fix are? Linking it here would help with context.

Reply:
Which issue? 

1. The issue around operator-sdk being unable to watch two resources of unstructured type.
2. The issue tracking the controller-runtime refactor that the operator-sdk is planning on.

If you would like 1 to be linked then we do have an issue for that but 2 is Jira stories and does not have a Github issue I believe. As Jira is not public I don't think it would help to add those links here IMO. 

Thoughts?

Reply:
One way or another, it would help to have enough information here so that if someone who doesn't have your prior knowledge looks at it, they'll be able to figure out if the workaround is still necessary.

Reply:
An issue was just created to track the operator-sdk refactor 😄